src/components/transactions/TxSigners/index.tsx

```diff
@@ -102,6 +106,22 @@ export const TxSigners = ({
   const isPending = useIsPending(txId)
   const wallet = useWallet()
   const { safe } = useSafeInfo()
+  const chainId = useChainId()
+  const [confirmations, setConfirmations] = useState<MultisigConfirmation[]>([])
+
+  useEffect(() => {
+    if (isMultisigDetailedExecutionInfo(detailedExecutionInfo)) {
+      setConfirmations(detailedExecutionInfo.confirmations)
+    }
+  }, [detailedExecutionInfo])
+
+  txSubscribe(TxEvent.SIGNATURE_INDEXED, ({ txId }) => {
```

Comment:
In the TxDetails we already have a bunch of conditions under which we will refresh the details. I would just add the txQueuedTag and txHistoryTag to that array:

```ts
const TxDetails = ({
  txSummary,
  txDetails,
}: {
  txSummary: TransactionSummary
  txDetails?: TransactionDetails // optional
}): ReactElement => {
  const chainId = useChainId()
  const { safe } = useSafeInfo()

  const [txDetailsData, error, loading] = useAsync<TransactionDetails>(async () => {
    return txDetails || getTransactionDetails(chainId, txSummary.id)
  }, [txDetails, chainId, txSummary.id, safe.txQueuedTag, safe.txHistoryTag])

// ...
}
```

That is how we also refresh the tx details for a singleTx in the `SingleTx`

Reply:
oh I didn't think about the tags! I'm going to give it a try

Reply:
I've implemented this way. Thank you for the valuable input!